Web/Program.cs

```diff
@@ -7,7 +7,7 @@
 services.AddDbContext<BookStoreDbContext>(options =>
 	options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"),
 		sqlOptions => sqlOptions.MigrationsAssembly(migrationAssembly)));
-
+builder.Services.AddScoped<DataSeeder>();
```

Comment:
The purpose of dependency injection is try to reuse the objects and optimize it. This class is only use once during app life time. You dont need to add it, just new instant